pyproject.toml

```diff
@@ -150,6 +150,16 @@ include-package-data = true
 where = ["src"]
 
 [tool.setuptools_scm]
+tag_regex = "^v(?P<version>20\\d{2}\\.\\d{1,2}\\.\\d{1,2})$"
```

Comment:
This tells `setuptools_scm` how to extract the "version" string.
- Require 4 digit year
- Allows 1 or 2 digit month and day strings (since PyPI and some other tools seem to default to 1 digit rather than allowing a leading zero)